virtual/docker/docker-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Virtual package for container engine"
+SLOT="0"
+KEYWORDS="amd64 arm64 ~loong ~riscv"
+
+RDEPEND="|| (
+	>=app-containers/docker-cli-23.0.0
+	app-containers/podman[wrapper(+)]
```

Comment:
With those two dependencies, the virtual should probably not be called `docker` but `docker-cli` (if there is a decision for this direction) to not be misleading. Thanks for your consideration.

Reply:
Actually based on testing this could be removed. `podman compose` works without the wrapper being present. And in most cases where "docker" is needed it is really the api where setting `DOCKER_HOST` to the podman socket solves it. Not sure if we want to support this or require users to build podman with the wrapper